app/ide-desktop/lib/dashboard/src/components/dashboard/DataLinkInput.tsx

```diff
@@ -1,23 +1,19 @@
 /** @file A dynamic wizard for creating an arbitrary type of Data Link. */
 import * as React from 'react'
 
-import Ajv from 'ajv/dist/2020'
-
 import SCHEMA from '#/data/dataLinkSchema.json' assert { type: 'json' }
 
 import type * as jsonSchemaInput from '#/components/JSONSchemaInput'
 import JSONSchemaInput from '#/components/JSONSchemaInput'
 
 import * as error from '#/utilities/error'
+import { AJV } from '#/utilities/validateDataLink'
```

Comment:
I need some help on the required TS code style here.

For this, I'm getting:
```
  10:1  error  No {} imports and exports  no-restricted-syntax
```

but then I did the following refactor:

Reply:
Overall what I was trying to solve was - we initialize AJV with 'known formats' in 2 places - which seems wrong as every time we add a new format, we'd need to ensure we keep these places consistent. Instead, I wanted to factor it out and have a single source of truth for known formats.